### PR TITLE
DIVOS-8860 if there's no box set width/height to 0

### PR DESCRIPTION
**Pull request by OliBravo**

### CODE REVIEW

**Review by cosi1 — APPROVED** [review]
LGTM

File: inst/ggtips/ggtips.js [hunk 1]
```diff
@@ -700,8 +700,8 @@ if (typeof jQuery === 'undefined') {
         left = offset.left;
         top = offset.top;
         var box = el.data('box');
-        var width = box.width;
-        var height = box.height;
+        var width = box === undefined ? 0 : box.width;
+        var height = box === undefined ? 0 : box.height;
```
Comment (jcubic) [inline inst/ggtips/ggtips.js:704]: When box is undefined something else is wrong with the code, did you check why the box is undefined?

Comment (jcubic) [reply inst/ggtips/ggtips.js:704]: The rest of the code may break if width/height is 0. This part of the code should never happen.
Can you share the case when box is undefined?

**Comment by jcubic** [comment]
Also can you use your GitHub email in commit?

**Comment by jcubic** [comment]
@cosi1 WTF why did you merge this? I've added review.

**Pull request merged**